osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -173,6 +181,7 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
                 time.sleep(5)
                 wait_time += 5
                 if wait_time > 30:
+                    logger.exception()
```

Comment:
Pass a message. (Here, `"Even after waiting for #{wait_time} #{time_unit_abbreviation}, the extraction order #{extraction_order_id} hasn't been created"` is probably more useful than the exception name.)